backend/tests/test_saves.py

```diff
@@ -0,0 +1,76 @@
+from saves import Saves
+import json, uuid
```

Comment:
Might be simpler, or easier to remember, to use the `PK` of the game for the id?

Reply:
i believe that's done now as I merged in your recommended schema.